src/doc/avrdude.texi

```diff
@@ -1456,10 +1456,12 @@ the MCU's ‘TXD’ line.
 Show help menu and exit.
 @end table
 
+@cindex @code{-x} jtag2updi
 @cindex @code{-x} serialupdi
+@item jtag2updi
 @item serialupdi
 
-The serialupdi programmer type accepts the following extended parameter:
+The jtag2updi and serialupdi programmer types accepts the following extended parameters:
```

Comment:
types accept (not accepts)

Reply:
Thanks! It's these things that always get me